projects/packages/jetpack-mu-wpcom/src/features/random-redirect/random-redirect.php

```diff
@@ -0,0 +1,102 @@
+<?php
```

Comment:
Copied from the file removed in https://github.com/Automattic/jetpack/pull/38310